source/reference/commands.rst

```diff
@@ -497,18 +497,24 @@ Commands
 	parameters accepted by this command.
 
 	- **cmd** [[String]]
+	- **shell_cmd** [String]: Shell command to use. If given overrides
+	  ``cmd`` and ignores ``shell``.
 	- **file_regex** [String]
 	- **line_regex** [String]
 	- **working_dir** [String]
 	- **encoding** [String]
 	- **env** [{String: String}]
-	- **path** [String]
-	- **shell** [Bool]
+	- **quiet** [Bool]: If ``True`` information less running about prints the
+	  command.
```

Comment:
This sentence makes no sense.

Reply:
Yeah, I'm with you on that. However, I didn't write it (and did not want to spend the time trying to figure out what it should say). I could delete it if that makes everyone happier? :smile:

Reply:
Change it to this: 

If True, don't print runtime information when the command fails or has a non-zero exit code.